🐍 lib_afni1D — Python 2→3

shnizzedy opened the pull request:
* Update [`lib_afni1D.py`](https://github.com/leej3/afni/blob/a9831f41b8f769a09b7599aa888612ef474e1338/src/python_scripts/afni_python/lib_afni1D.py) and its dependencies to Python 3
* Update [`setup.py`](https://github.com/shnizzedy/afni/tree/8ccdbb81ca294038c20740fc2152f54595a94a44/src/python_scripts/setup.py)
    * URL (from `git+https://github.com/leej3/template_making.git` to `git+https://github.com/afni/afni.git@master#egg=afni-python&subdirectory=src/python_scripts`)
    * version (from `0.0.1` to `0.0.2`)

---

I saw https://github.com/afni/afni/issues/130 and https://github.com/afni/afni/pull/154, so I pointed this PR at [`leej3:change_python_installation_approach`](https://github.com/leej3/afni/tree/change_python_installation_approach) even though I forked from [`afni:master`](https://github.com/afni/afni). If you suggest I point a PR there in addition to or instead of this one, I'd be glad to.

shnizzedy committed:
:snake: Python 2 → 3

for lib_afni1D and its dependencies

shnizzedy committed:
:rocket: Update pip installable URL and :bookmark: bump version to 0.0.2

leej3 commented:
Thanks @shnizzedy. Could you set up the pull request to afni/master? 

I will be working on merging #154 later in the week.

shnizzedy committed:
:rewind: Revert ":snake: Python 2 → 3"

This reverts commit a9831f41b8f769a09b7599aa888612ef474e1338.

shnizzedy committed:
:snake: Python 2 → 3

for lib_afni1D and its dependencies

(:ok_hand: a9831f4 with fewer whitespace & style changes)
